www/api/index.php

```diff
@@ -145,6 +145,18 @@ function if_is_admin($callback, $elseCallback=null) {
     ok();
 });
 
+$app->get('/v3_reachability', function() use ($app) {
```

Comment:
```suggestion
$app->get('/status-v3', function() use ($app) {
```
how about this name (i think we use dashes instead of underscores in url routes typically)